onnxruntime/python/tools/symbolic_shape_infer.py

```diff
@@ -2389,6 +2389,29 @@ def parse_arguments():
         type=int,
         default=0,
     )
+    parser.add_argument(
+        "--save_as_exteranl_data",
+        help="Saving an ONNX model to external data",
+        action="store_true",
+        default=False,
+    )
+    parser.add_argument(
+        "--all_tensors_to_one_file",
+        help="Saving all the exteranl data to one file",
```

Comment:
exteranl is a typo

Reply:
The onnx api has this option and probably it is to provide flexibility.
https://github.com/onnx/onnx/blob/main/docs/ExternalData.md#converting-and-saving-an-onnx-model-to-external-data